yb-voyager/cmd/common.go

```diff
@@ -106,7 +106,7 @@ func getMappingForTableNameVsTableFileName(dataDirPath string, noWait bool) map[
 
 	pgRestorePath, err := srcdb.GetAbsPathOfPGCommand("pg_restore")
 	if err != nil {
-		utils.ErrExit("could not get absolute path of pg_restore command: %v", pgRestorePath)
+		utils.ErrExit("could not get absolute path of pg_restore command: %w", err)
```

Comment:
```
func ErrExit(formatString string, args ...interface{}) {
	formatString = strings.Replace(formatString, "%w", "%s", -1)
	fmt.Fprintf(os.Stderr, formatString+"\n", args...)
	log.Errorf(formatString+"\n", args...)
	atexit.Exit(1)
}
```
The `Fprintf()` does not support `%w` format specifier. `%w` works only in the `fmt.Errorf()`. If you need to print error objects, go for `%s`.

Reply:
Oh yes, thanks!